packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/auth/ApiKeyAuth.kt

```diff
@@ -0,0 +1,12 @@
+package io.realm.kotlin.mongodb.auth
+
+import io.realm.kotlin.types.ObjectId
+
+public interface ApiKeyAuth {
```

Comment:
Class and all methods are missing documentation, including which errors each can throw.